### ISSUE TITLE
Expose a json API to get current standings or control who I'm spectating

### ISSUE THREAD

**Issue by MiniGod** [ManiaJS/plugins#18]
Expose an API to get public data about the ongoing race. Could be loaded on websites to show live stats. Maybe even support EventStream (SSE)?

Public data like current standings, map list, player list, chat, could be easy as it's read-only and public.

However, we could expand from this and let admins control ManiaJS -- or even the server it self through ManiaJS -- using the same API. 

It could also be great for streamers to be able to for example have the player list in their browser and switch the players there. I'm no streamer so I don't know if that would work well.

**Comment by tomvlk** [comment]
This one is gold, needs to be in there for sure! 

Maybe we can add this to the core itself with enable/disable to make sure we have full power. 
Or maybe even make this work in a separate process. (with cluster) so it won't directly be exposed to the performance (keep in mind for any flooding/ddos attacks, it can crash the entire server..).

**Comment by MiniGod** [comment]
I was thinking about the flooding and dos attacks as well, but I think we shouldn't put too much effort into that apart from warning people that it is a "expert" feature and should be put behind a proxy of some sort.

**Comment by tomvlk** [comment]
Yes, behind reverse proxy will be better, but best is to use it internally.
